homework/index.js

```diff
@@ -28,23 +28,56 @@
     });
     return elem;
   }
+  function renderRepoDetails(table, title, repoText, article, repoHref = '') {
+    const tr = createAndAppend('tr', table);
+    if (article === 'a') {
+      createAndAppend('th', tr, {
+        text: title,
+      });
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+      createAndAppend(article, tr, {
+        text: repoText,
+        href: repoHref,
+      });
+    } else {
+      createAndAppend('th', tr, {
+        text: title,
+      });
+      createAndAppend(article, tr, { text: repoText });
+    }
+  }
+
+  function sortName(a, b) {
+    return a.name.localeCompare(b.name);
   }
```

Comment:
The single-line function is not really needed. You can use an arrow as shown below.